apps/angularlogos/src/app/metacheck/metacheck.component.html

```diff
@@ -1,4 +1,8 @@
 <h2>Logos without metadata</h2>
 
-<strong>Count:</strong> {{ (logosWithoutMetadata$ | async)?.length }}
-<pre>{{ fullLogoMetadata$ | async | json }}</pre>
+<ng-container *ngTemplateOutlet="template; context: logosWithoutMetadata$ | async"></ng-container>
```

Comment:
Why?! The template outlet introduces too much complexity IMO. Why not:

```
<ng-container *ngIf="logosWithoutMetadata$ | async as data">
    <strong>Count:</strong> {{ data.count }}
    <pre>{{ data.logos | json }}</pre>
</ng-container>
```

Reply:
Hmm didn't try it with an `ng-container` but with an `ng-template` and that one did not work. Will try this one.